CriteoPublisherSdk/Sources/Standalone/CRBannerView.m

```diff
@@ -165,6 +174,11 @@ - (void)loadAdWithBid:(CRBid *)bid {
     return;
   }
 
+  if (self.adUnit.adUnitId == nil && [bid.adUnit isKindOfClass:[CRBannerAdUnit class]]) {
```

Comment:
On `self.adUnit.adUnitId == nil` and ``self.adUnit = bid.adUnit`:

Why do we need to register the ad unit of the bid ? I don't see any reason why we should keep track of it. Moreover, this means that you introduce a temporal dependency in `loadAdWithBid` calls.

Given this scenario:
- load successfully bid1 from adUnit1
- display bid1 -> good
- load successfully bid2 from adUnit2
- display bid2 -> not good as the frame same might not match the new ad

Hence I'm not sure that we need to store the adUnit. On the contrary, I think that this make the flow more complicated.

Reply:
Remove setting the `adUnit` locally as it is useless.